### PR TITLE
Add storybook-addon-performance

### PR DESCRIPTION
**Pull request by PavelVanecek**
## Description

I am not immediately certain of the usefulness of this addon since it reports renders in seconds and that will be different in each run and each computer; and it reports all of recharts to have 1 DOM element which makes me suspicious it's not seeing the whole picture.

## Related Issue

https://github.com/recharts/recharts/issues/3437

## How Has This Been Tested?

Storybook

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a storybook story or extended an existing story to show my changes
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by nikolasrieble** [comment]
Do you see a reason to not add this for now? Do you think https://github.com/recharts/recharts/pull/3829 is sufficient instead? 

Having two ways to measure performance might add more noise than value. What do you think?

**Comment by PavelVanecek** [comment]
I think both tools are valuable because they offer different information to different audience.

One aimed for contributors, repeatable and reliable, but more difficult to setup and use.

Another for general public, zero setup and quick to give an idea of what recharts can do and cannot do performance wise.

**Comment by ckifer** [comment]
@PavelVanecek does this slow down storybook more than it already is?

I feel we're bogging things down a bit - if I can't run the storybook locally without my computer exploding then I think we might need to rethink adding this

**Comment by PavelVanecek** [comment]
The storybook starts at the same speed as before. The plugin has a button "start measurement" and only when you switch to the "Performance" tab and push the button your computer starts exploding.

**Comment by ckifer** [comment]
lol, gotcha. If you fix the merge conflicts we can merge this then

**Comment by PavelVanecek** [comment]
I resolved and pushed but github still shows conflicts ಠ_ಠ

**Comment by ckifer** [comment]
Weird, why does it pick up all the other PRs changes

**Comment by ckifer** [comment]
Nice, closing and re-opening reset it

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ _Before_ submitting a pull request, please make sure the following is done…
 ## <a name="code"></a>Code Guide
 
 Our linter will catch most styling issues that may exist in your code.
-You can check the status of your code styling by simply running: `npm run lint`
+You can check the status of your code styling by running: `npm run lint`
```
Comment (nikolasrieble) [inline CONTRIBUTING.md:73]: I love small language cleanups, thank you!

**Review by nikolasrieble — APPROVED** [review]
Lets go!

**Pull request merged**